packages/server-bridge/src/bridge.js

```diff
@@ -353,6 +374,34 @@ function init({
       });
   }
 
+  async function onHttpRequest(req, res) {
+    const { body: { body }, params, user } = req;
+    const queueName = join(split(params[0], '/'), ':');
+    const initSession = startHttpSession(log, stellarRequest.source, req);
```

Comment:
need to add other session vars via call to `callHandlersSerially` like in `init`

Reply:
If that's true I need to change quite a bit in `callHandlersSerially`
because the first handler it calls is
```
function assignClientToSession({ log, source, socket, session }) {
  return assign(session, { client: new WebsocketTransport(socket, source, log, true) });
}
```